<launch>
  <node name="cmt_node" pkg="cmt" type="cmt_node" output="screen"/>
  <node name="kf_main" pkg="cmt" type="kf_main.py" output="screen"/>
  <node pkg="rqt_reconfigure" name="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>
  <node pkg="cmt" name="server_node" type="server_node.py" output="screen"/>
</launch>
